CryptoPrices/Sources/MyCoin/Sources/MyCoin/TimeFrameSectionView/TimeFrameSection.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  TimeFrameSection.swift
+//  MyCoin
+//
+//  Created by Khanh on 04/01/2023.
+//
+
+import Styleguide
+import SwiftUI
+
+struct TimeFrameSection: View {
+    
+    @State var selected: TimeFrameItem = .init(timeFrame: .oneDay)
+    
+    private var timeFrameList: [TimeFrameItem] = [
+        .init(timeFrame: .oneDay),
+        .init(timeFrame: .oneWeek),
+        .init(timeFrame: .oneMonth),
+        .init(timeFrame: .oneYear),
+        .init(timeFrame: .fiveYears)
+    ]
```

Comment:
Since the list won't change throughout the application runtime, I think we can turn it into `let` 😁

Reply:
Good catch 🙌 . I fixed it at https://github.com/nimblehq/nimble-crypto-ios/pull/86/commits/e096d963ba508001f5aae3513b0af1dc28571083.